packages/tools/e2e-tests/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["node"],
+    "rootDir": "./",
+    "target": "ES2020",
+    "skipLibCheck": true,
+    "noEmit": true,
+    "incremental": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "declaration": false, // fixes pnpm issue: https://github.com/microsoft/TypeScript/issues/42873
+    "declarationMap": false,
+    "jsx": "preserve",
+    "allowJs": true
+  },
+  "include": ["/**/*.ts", "/**/*.tsx"],
```

Comment:
This workspace also has its files in `src` dir? You can omit this `include`? No need to ignore that `playwright.config.ts` and `node_modules`.

Reply:
I'll change this. I moved all folders in to a src directory fairly late on because I felt it was quite messy otherwise.  The above was required to get it working previously.